### PR TITLE
add check script for reflector

### PR DESCRIPTION
**Pull request by cleverhu**

### CODE REVIEW

**Comment by Iceber** [comment]
@cleverhu https://github.com/clusterpedia-io/clusterpedia/pull/210 has been merged, you could rebase code.

File: hack/verify-vendor.sh [hunk 1]
```diff
@@ -47,3 +47,14 @@ else
...
 fi
+
+reflector=0
+diff vendor/k8s.io/client-go/tools/cache/reflector.go pkg/synchromanager/clustersynchro/informer/.reflector.go || reflector=$?
+
+if [[ $reflector -eq 0 ]]
+then
+  echo "${DIFFROOT} up to date."
+else
+  echo "${DIFFROOT}, the file 'reflector.go' in vendor has been changed, please update the '.reflector.go' and 'reflector.go' in the pkg/synchromanager/clustersynchro/informer"
```
Comment (Iceber) [inline hack/verify-vendor.sh:58]: When it failes, printing ${DIFFROOT} variables seems unnecessary

File: hack/verify-vendor.sh [hunk 2]
```diff
@@ -47,3 +47,14 @@ else
   echo "${DIFFROOT}, 'go.mod' or 'go.sum' is out of date. Please run 'make vendor'"
```
Comment (Iceber) [inline hack/verify-vendor.sh:47]: This place too, `${DIFFROOT}` looks very redundant

File: hack/verify-vendor.sh (hunk 1 above)
Comment (cleverhu) [reply hack/verify-vendor.sh]: The path of 'reflector.go' is omitted because i think the error message is clear.

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**